### PR TITLE
shalinee | TEST | correct UI of existing responsive module

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 1]
```diff
@@ -94,7 +94,7 @@
                                     </div>
                                 </div>
                                 <!-- On company class will show only if we are on company mode and branches length will be less than equal 2 -->
-                                <div class="pull-left header-left" [ngClass]="{'on-company': currentCompanyBranches?.length <= 2 }">
+                                <div class="pull-left header-left" [ngClass]="{'on-company': currentCompanyBranches?.length >= 2 }">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.html:97]: Please check this condition change

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.scss [hunk 2]
```diff
@@ -157,3 +157,8 @@ label {
         }
     }
 }
+@media (max-width:767px) {
+    .custom-tabs::ng-deep .nav-tabs{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.scss:161]: ng-deep is deprecated, avoid usage

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/contact.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/contact.component.html:97]: It is working correct

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.scss]: Fixed

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**